physical/limit.go

```diff
@@ -45,3 +46,10 @@ func (node *Limit) Materialize(ctx context.Context, matCtx *MaterializationConte
 func (node *Limit) Metadata() *metadata.NodeMetadata {
 	return metadata.NewNodeMetadata(metadata.BoundedFitsInLocalStorage, node.Source.Metadata().EventTimeField())
 }
+
+func (node *Limit) Visualize() *graph.Node {
+	n := graph.NewNode("Limit")
+	n.AddChild("limit", node.LimitExpr.Visualize())
+	n.AddChild("source", node.Source.Visualize())
```

Comment:
I think that mostly "source" child is added in the first place, so maybe swap those 2 lines

Reply:
I'm trying to write them in an intuitive order which is based on how they're used.

If you have a limit, you first check it, and then look into the source.

If you have a map, you first get a record and then transform it.